src/commands/audio/start.ts

```diff
@@ -59,15 +61,26 @@ export class UserCommand extends ArimaCommand {
 			return sendError(interaction, UserCommand.errorDescriptors[result.error]);
 		}
 
-		const game = new StandardGame({
+		const gameType = (interaction.options.getString('gametype') as GameType) ?? GameType.Standard;
+		const gameData = {
```

Comment:
If you're going to make this separately (which you don't have to depending on whether you change the switch statement below to a different form of something we discussed in Discord), you'll need to type it
```suggestion
		const gameData: GameData = {
```